scala3doc/src/dotty/dokka/translators/ScalaSignatureProvider.scala

```diff
@@ -49,6 +49,8 @@ object ScalaSignatureProvider:
         extensionSignature(extension, builder)
       case method: DFunction if method.kind.isInstanceOf[Kind.Given] =>
         givenMethodSignature(method, builder)
+      case exprt: DFunction if exprt.kind == Kind.Exported =>
```

Comment:
Do we need that `case` and  `exportedMethodSignature` since we handle it as `methodSignature` anyway ?

Reply:
Can delete, but thought it would be more reasonable to state that explicitly